.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'Bug: '
+labels: bug
+assignees: ''
+
+---
+
+**System Information**
```

Comment:
I think we should add a comment here that if the bug is a security issue, it should not be opened and that they should check SECRUTIY.md instead

Reply:
Thanks for noting that!

We will have to add it once we have both relevant PRs merged as the Security Policy is currently absent until #339 is merged.